packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md

```diff
@@ -162,257 +163,334 @@ Continue with the OIN Wizard and specify your supported SSO flows:
+    {style="list-style-type:lower-alpha"}
+    1. Open an incognito browser window to sign in to Okta.
+    1. Sign in to Okta as the new test user.
+    1. Select the app tile.
+    1. Wait for confirmation that the new test user signed in and was provisioned in your app (you're responsible to verify this step).
+
+1. Verify that the test user signed in to your app with the correct first name, last name, and email attributes.
+
+    > **Note:** You can go back to the OIN Submission Tester window and expand the test case to view network traffic details for this test run. The **NETWORK TRAFFIC** tab contains API calls to Okta with the test user details in the request payload.
+
+1. Select **The user successfully signed in to your app** in the upper-right **Test in progress** dialog to confirm that the JIT provisioning IdP flow test passed.
+1. Click **Continue** from the **Test in progress** dialog to sign out of your app.
+
+    The OIN Submission Tester executes the following steps after you click **Continue**:
+    1. Sign out of the app and close the incognito browser window.
+    {style="list-style-type:lower-alpha"}
+    1. Unassign the test user from the app instance in Okta.
+    1. Deletes the test user from Okta.
+    1. Records the test run result and timestamp in the OIN Submission Tester.
+    1. Redirected you back to the OIN Submission Tester.
+
+1. Click the **JIT provisioning (w/ IdP flow)** expand icon (![chevron-down expand icon](/img/icons/chevron-down-odys.svg)) to view the test steps and network traffic details for the test run.
+
+> **Note:** The test user account created in your app from JIT provisioning persists after the JIT provisioning test. The OIN Submission Tester only removes the temporary test user account from your Okta org. It's your responsibility to manage the JIT test user accounts in your app.
+
+##### Run the JIT provisioning with SP flow test
+
+You're only required to pass one JIT provisioning test case to submit your integration. The OIN Submission Tester includes the **JIT provisioning (w/ SP flow)** test case if you support JIT and only the SP flow. If your integration supports JIT, IdP, and SP flows, then a successful **JIT provisioning (w/ IdP flow)** test is sufficient for submission.
+
+Similar to the [JIT provisioning with IdP flow test](#run-the-jit-provisioning-with-idp-flow-test), the OIN Submission Tester creates a temporary Okta test user account for you to verify that JIT provisioning was successful. The Tester then removes the test user account from Okta to complete the test.
 
-To update a submitted integration:
+To run the JIT provisioning with SP flow test, follow the same steps in [Run the JIT provisioning with IdP flow test](#run-the-jit-provisioning-with-idp-flow-test). The difference between the JIT provisioning with SP and IdP is that the OIN Submission Tester opens an incognito browser window to sign in to your app first instead of to Okta.
```

Comment:
```suggestion
To run the JIT provisioning with SP flow test, follow the same steps in [Run the JIT provisioning with IdP flow test](#run-the-jit-provisioning-with-idp-flow-test). The difference between the JIT provisioning with SP and IdP tests is that wthe OIN Submission Tester opens an incognito browser window to sign in to your app first instead of to sign in to Okta.
```

Reply:
Thanks! I'm going to update this in another commit.